sdk/cloudmachine/Azure.CloudMachine/api/Azure.CloudMachine.net8.0.cs

```diff
@@ -1,18 +1,5 @@
 namespace Azure.CloudMachine
 {
-    public partial class AIFoundryClient : Azure.Core.ClientWorkspace
```

Comment:
Let's not remove it now. We need it for the first demo.

Reply:
I have already released an alpha version with what we have in the main branch today, which includes the `AIFoundryClient` type.  

I also created Demo-1 here:  
[https://github.com/ShivangiReja/CloudMachine_AIFoundry_Demo1/blob/master/Program.cs](https://github.com/ShivangiReja/CloudMachine_AIFoundry_Demo1/blob/master/Program.cs)  

Can we remove `AIFoundryClient` now?

Reply:
Discussed offline: we will keep AIFoundryClient for now.